Packs/WithSecure/Playbooks/playbook-WithSecure_EDR_Incident_Response.yml

```diff
@@ -0,0 +1,417 @@
+id: WithSecure EDR Incident Response
```

Comment:
please add a photo of the playbook in the doc_files folder and add it to the readme of the playbook as well